charts/kong/README.md

```diff
@@ -768,6 +768,7 @@ section of `values.yaml` file:
 | gatewayDiscovery.adminApiService.namespace | The namespace of the Kong admin API service (for more details see [gatewayDiscovery section][gd_section])                                                | `.Release.Namespace`               |
 | gatewayDiscovery.adminApiService.name      | The name of the Kong admin API service (for more details see [gatewayDiscovery section][gd_section])                                                     | ""                                 |
 | konnect.enabled                            | Enable synchronisation of data plane configuration with Konnect Runtime Group                                                                            | false                              |
+| konnect.runtimeGroupID                     | Deprecated: Konnect Runtime Group's unique identifier.                                                                                                   |                                    |
```

Comment:
Should we mention that Konnect does not use the "Runtime Group" concept (replaced by "Control Plane" now)?

Reply:
I don't think it has been completely replaced yet.

And do we need to provide too much content here?